src/oc/storage/api/boards.clj

```diff
@@ -244,7 +254,8 @@
   (timbre/info "Deleting board:" slug "of org:" org-slug)
   (if-let* [org (:existing-org ctx)
             board (:existing-board ctx)
-            _delete-result (board-res/delete-board! conn (:uuid board))]
+            entries (entry-res/list-all-entries-by-board conn (:uuid board))
```

Comment:
Above causes this code duplication (query for entries.)  Moving this code into the board resources module will remove the need to query for related entries each time you delete a board.

Reply:
I can't use the entry resource namespace from the board resource, it's causing a circular dependency.
This is the simplest way i found to make it work